javascript-local-storage/data.js

```diff
@@ -0,0 +1,35 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+if (previousTodosJSON !== null) {
+  todos = JSON.parse(previousTodosJSON);
+}
+
+function eventFunction(event) {
+  var todosJSON = JSON.stringify(todos);
+  localStorage.setItem('javascript-local-storage', todosJSON);
+}
+
+window.addEventListener('beforeunload', eventFunction);
+
+// function removeItem(event) {
+
+// }
+
+// .addEventListener('dblclick')
+
+var listContainer = document.querySelector('#todo-list');
+
+function hideFunction(event) {
+  // console.log(event.target);
+  // console.log(event.target.tagName);
+
+  if (event.target.tagName === 'LABEL' || event.target.tagName === 'INPUT') {
+    var closestAncestor = event.target.closest('.list-group-item');
+    closestAncestor.remove();
+  }
+}
+
+listContainer.addEventListener('dblclick', hideFunction);
```

Comment:
This can be deleted, all this is handled by the code that was pre-written for the exercise.